src/main/resources/db/migration/all/20230913145208__add_mi_reports_columns_to_temporary_accommodation_application.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE temporary_accommodation_applications ADD COLUMN is_registered_sex_offender BOOLEAN;
```

Comment:
What's the behaviour for setting the default value here? Do we need DEFAULT NULL to make sure old referrals don't have boolean values of false?